README.md

```diff
@@ -1,94 +1,27 @@
+## Intro
 
-## The planning and roadmap process
+Welcome to SIG-Release, home of release and feature management. If you are new to this repository, you might find the getting started guide useful. If you are interested in the current state of planning, have a look at our Release Planning Board.
 
-We follow a coordinated approach to plan improvements of Eclipse Tractus-X. see also [Open Planning and Refinement](docs/open-refinement-and-planning.md).
+[Catena-X Standards](https://catena-x.net/de/standard-library) are the foundation for certifying any software product operating in the Catena-X data space. Those standards are the binding reference to obtain a valid certificate.
 
-While every repository in the [eclipse-tractusx](https://github.com/eclipse-tractusx) GitHub organization
-has its own issue management, the [Release Planning Board](https://github.com/orgs/eclipse-tractusx/projects/26)
-is used to align the overarching Tractus-X releases.
+To make adopting the latest [Catena-X Standards](https://catena-x.net/de/standard-library) easier, reference implementations are provided through the Eclipse Tractus-X Project
 
-### How can I get involved
+## Release Cycle - major vs. minor release
 
-In case you experienced a bug, unexpected behaviour, or you want to propose enhancements to Eclipse Tractus-X,
-feel free to use one of the provided [issue templates](https://github.com/eclipse-tractusx/sig-release/issues/new/choose) and describe your request.
-Please be aware, that not every feature request can be integrated and that we also cannot treat every issue with the highest priority.
+Tractus-X operates on a quarterly release cycle. Each quarter, a new [Tractus-X release](docs/tractus-x-release.md) following [CalVer](https://calver.org) is released.
 
-Every Release planning will be kicked off by two public alignment sessions. The dates and further details will be shared via
-[tractusx-dev](https://accounts.eclipse.org/mailing-list/tractusx-dev) mailing list.
-In addition to that, you can also find public meetings and info about how to join on our
-[Open Meetings](https://eclipse-tractusx.github.io/community/open-meetings) community page. 
-Issues or bug reports, that should be discussed in these meetings, have to be opened prior to the meeting via
-our [issue templates](https://github.com/eclipse-tractusx/sig-release/issues/new/choose).
+Each Tractus-X release contains [multiple products](docs/product_release.md).
 
-### What can I expect
+Once per year, Tractus-X releases a major release, making the remaining three releases minor releases. A major release may contain critical breaking changes that have a major impact on data space participants, such as changes to enablement services. A minor release contains backward compatible functionality. In addition, patch versions provide backward compatible bug and security fixes.
```

Comment:
As I am working in the release management group in the association, the impact of the breaking change release thing is currently detailed. The clear requirement is, that the dataspace runs smoothly and companies can update in a defined grace period. This is defining "breaking changes" on a conceptual level. It has to be detailed, what this means for concrete developments. It is not possible to have breaking changes on the dataspace level, i.e., in the interactions companies have with other companies and especially with the operating company. So this statement here will break this requirement. Please wait until clarification is agreed upon.

Reply:
fine for me... @lgblaumeiser lets sync after the PR is merged. Would be happy to contribute here

Reply:
Is ok, as the impact of the sentence is still quite far in the future.

Reply:
@lgblaumeiser i changed sooo much, pls let us merge it now and finetune it next week with @stephanbcbauer